test_vm/tests/replica_update_test.rs

```diff
@@ -546,7 +550,7 @@ fn upgrade_bad_post_dispute() {
     let store = &MemoryBlockstore::new();
     let policy = Policy::default();
     let (v, sector_info, worker, miner_id, deadline_index, partition_index, _) =
-        create_miner_and_upgrade_sector(store);
+        create_miner_and_upgrade_sector(store, false);
```

Comment:
Why not test v2 for these tests?

Reply:
because it is unrelated to the changes, I can add these tests if we want to.